requirements.txt

```diff
@@ -8,5 +8,5 @@
 # agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
+git+https://github.com/shotgunsoftware/python-fernet.git@v1.0.1-python2#fernet
 websocket_client == 0.56.0
-git+https://github.com/shotgunsoftware/python-fernet.git@v1.0.1-python2#fernet
```

Comment:
I'm not sure why the line endings moved this up a line?
It doesn't seem like it would be an issue though.

Reply:
I think it's the pre-commit hook which reordered the requirements.